video/users/client-initialization/example.js

```diff
@@ -0,0 +1,11 @@
+// Create an AccessManager which can manage the expiration and lifecycle of
```

Comment:
we should try to keep the file names the same as the directory, almost sure that is a requirement of the CMS